tools/libboard/board_n5010/board_n5010.c

```diff
@@ -0,0 +1,266 @@
+// Copyright(c) 2019-2020, Intel Corporation
```

Comment:
Could we add some unit tests for this new board module under tests/board/?

Reply:
Done